wget/wget.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: wget
+spec:
+  params:
+   - name: url
+     description: URL to wget
+     type: string
+     default: ""
+   - name: diroptions
+     description: options of wget to change file location
+     type: array
+     default: []
+   - name: dir
+     description: options of dir
+     type: string
+     default: "$(workspaces.myworkspace.path)"
+   - name: options
+     description: options of wget
+     type: array
+     default: []
+   - name: filename
+     description: name of file we want to change
+     type: string
+     default: ""
+  workspaces:
+   - name: myworkspace
```

Comment:
small nit: suggest we use a more descriptive name than `myworkspace`. How about `wget-workspace`?